templates/appliance/shared/_verify-checksums.html

```diff
@@ -0,0 +1,15 @@
+<p>
+  You can
+  <span class="p-contextual-menu--left">
+    <a href="#" class="p-contextual-menu__toggle" data-trigger="click" aria-controls="#menu-4" aria-expanded="false" aria-haspopup="true">verify your download</a>
```

Comment:
Is there an id called menu-4?

Reply:
in the next line? `<span class="p-contextual-menu__dropdown" id="menu-4" aria-hidden="true" aria-label="Verification instructions:" style="width: 600px; max-width: 70vw; padding: 1rem;">`